src/txn/txn_rollback_to_stable.c

```diff
@@ -225,13 +225,6 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
             WT_ERR(__wt_buf_set(session, &full_value, hs_value->data, hs_value->size));
         }
 
-        /*
-         * Verify the history store timestamps are in order. The start timestamp may be equal to the
-         * stop timestamp if the original update's commit timestamp is out of order.
-         */
-        WT_ASSERT(session,
-          (newer_hs_ts == WT_TS_NONE || hs_stop_ts <= newer_hs_ts || hs_start_ts == hs_stop_ts));
```

Comment:
We've fought pretty hard to keep this assert in - why isn't it valid any more?

Reply:
I moved this change of discussion into another ticket, so removed here.